src/org/thoughtcrime/securesms/jobs/PushDecryptJob.java

```diff
@@ -639,6 +645,16 @@ private void handleTextMessage(@NonNull MasterSecretUnion masterSecret,
 
     if (smsMessageId.isPresent() && !message.getGroupInfo().isPresent()) {
       threadId = database.updateBundleMessageBody(masterSecret, smsMessageId.get(), body).second;
+    } else if (body.length() >= 19 && body.substring(0, 19).equals("{\"type\": \"reaction\"")) {
+      threadId = DatabaseFactory.getThreadDatabase(context).getThreadIdFor(recipient);
+      ReactionsHandler handler = new ReactionsHandler(context);
+      ObjectMapper mapper = new ObjectMapper();
+      Map<String,String> map = mapper.readValue(body, Map.class);
+
+      //add reaction to db
```

Comment:
comment seems arbitrary, this level of documentation is not in the rest of the code.